index.html

```diff
@@ -10,8 +10,18 @@
     <header>
       <div id="welcome">Welcome</div>
       <nav id="nav">
```

Comment:
- [ ]  Please give padding to the sides of hamburger button in order to match the design better

Yours:
![image](https://user-images.githubusercontent.com/62705947/232614645-7930e172-700f-4b33-94eb-58c325642356.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/232614597-278d9b26-9e61-40cc-93a7-6dca74f4329e.png)